doc/api/esm.md

```diff
@@ -241,10 +241,16 @@ There are four types of specifiers:
 Bare specifiers, and the bare specifier portion of deep import specifiers, are
 strings; but everything else in a specifier is a URL.
 
-Only `file://` URLs are supported. A specifier like
+Only `file:` and `data:` URLs are supported. A specifier like
 `'https://example.com/app.js'` may be supported by browsers but it is not
 supported in Node.js.
 
+`data:` URLs only resolve _Bare specifiers_ for builtin modules and
+_Absolute specifiers_. Resolving _Relative specifiers_ will not work because
+`data:` is not a [special scheme]. For example, attempting to load `./foo`
```

Comment:
Doesn't the markup have to be `[special scheme][]`?

Reply:
this works in GFM at least? idk, we can change it if desired

Reply:
I found over 2000 examples of using the trailing `[]`, and couldn't find any counter examples (though maybe my regex-fu was insufficient). I'd prefer to stay consistent.

Reply:
fixed (GH seems to be taking a bit of time updating the preview though)